File: examples/javascript/create-memory.mjs
import { memside } from "./memside-client.mjs";

const created = await memside("/memories", {
  method: "POST",
  body: JSON.stringify({
    text: "Packing checklist\nBring a charger and a reusable bottle.",
    type: "note",
    sensitivity: "private",
    tags: ["travel", "checklist"],
  }),
});

console.log(JSON.stringify(created, null, 2));

File: examples/python/create_memory.py
import json

from memside_http import api_request


created = api_request(
    "POST",
    "/memories",
    body={
        "text": (
            "Packing checklist\n"
            "Bring a charger and a reusable bottle."
        ),
        "type": "note",
        "sensitivity": "private",
        "tags": ["travel", "checklist"],
    },
)

print(json.dumps(created, indent=2))
